Assets/MixedRealityToolkit/Common/Scripts/CrossPlatform/TypeUtils.cs

```diff
@@ -6,7 +6,7 @@
 using System.Reflection;
 #endif
 
-namespace MixedRealityToolkit.Common
```

Comment:
Why has this namespace changed, as it is located in the "Common" folder?

Reply:
I believe this is also needed by SWIG files.
@keveleigh we should atleast write a comment in these files explaining why they don't have a namesapace.

Reply:
My Point @NeerajW is that (agreeing with @StephenHodgson ) is that we should fix the generator (in the future) rather than dirty the API.